sample/src/main/java/com/afollestad/materialdialogssample/MainActivity.kt

```diff
@@ -641,6 +642,19 @@ class MainActivity : AppCompatActivity() {
       }
     }
 
+    colorChooser_primaryColorsAndCustomValues.setOnClickListener {
+      MaterialDialog(this).show {
+        // title does not look good in landscape mode
```

Comment:
Wonder if there's something that can be done about that?

Reply:
No idea... The tabs could be disabled in landscape mode or the title could be ignored in landscape mode, both not that intuitive though... Currently it's the user's decision, he could provide a title in portrait mode only e.g....